themes/default/content/docs/guides/continuous-delivery/azure-devops.md

```diff
@@ -31,6 +31,21 @@ the steps outlined in the sample YAML file below to the Visual Designer as well.
 - A git repo with your Azure DevOps project set as the remote URL.
     - To learn more about how to [create a git repo in your DevOps project](https://docs.microsoft.com/en-us/azure/devops/organizations/projects/create-project?view=vsts&tabs=new-nav).
 
+## Command Summary
```

Comment:
should the table in this section, list required parameters first? i think that's our general pattern, but happy to break from it if there's a specific reason to.

Reply:
Fixed.